store/src/main/java/org/apache/rocketmq/store/config/MessageStoreConfig.java

```diff
@@ -81,8 +80,8 @@ public class MessageStoreConfig {
     /**
      * 1. Register to broker after (startTime + disappearTimeAfterStart)
      * 2. Internal msg exchange will start after (startTime + disappearTimeAfterStart)
-     *  A. PopReviveService
```

Comment:
No need to modify indentation here

Reply:
> No need to modify indentation here

This is caused by tool formatting， I will fix it and resubmit later